src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocvariable/InputJavadocVariableNoJavadoc2.java

```diff
@@ -0,0 +1,124 @@
+package com.puppycrawl.tools.checkstyle.checks.javadoc.javadocvariable;// violation
```

Comment:
Space before `//` and empty line after package declaration. This mistake is in lot of files. Please go through them and correct them all.

Reply:
Not done. [InputJavadocVariableNoJavadoc2.java](https://github.com/checkstyle/checkstyle/blob/fb62f8f570707d313d82ebe22d65f209f6892a2e/src/test/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/javadocvariable/InputJavadocVariableNoJavadoc2.java) and InputJavadocVariableNoJavadoc4.java are missed.

Reply:
Resolved.